Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/ElementStateProvider.cs

```diff
@@ -9,12 +9,14 @@ namespace Aquality.Selenium.Core.Elements
     public class ElementStateProvider : IElementStateProvider
     {
         private readonly By elementLocator;
+        private readonly LogElementState logElementState;
 
-        public ElementStateProvider(By elementLocator, IConditionalWait conditionalWait, IElementFinder elementFinder)
+        public ElementStateProvider(By elementLocator, IConditionalWait conditionalWait, IElementFinder elementFinder, LogElementState logElementState)
         {
             this.elementLocator = elementLocator;
             ConditionalWait = conditionalWait;
             ElementFinder = elementFinder;
+            this.logElementState = logElementState;
```

Comment:
Why do we need it?

Reply:
In order to avoid passing 4 parameters: ElementType, Name, LocalizationManager, LocalizedLogger